common/lib/xmodule/xmodule/library_content_module.py

```diff
@@ -434,6 +434,18 @@ def validate(self):
         validation = super(LibraryContentDescriptor, self).validate()
         if not isinstance(validation, StudioValidation):
             validation = StudioValidation.copy(validation)
+        library_tools = self.runtime.service(self, "library_tools")
+        if not (library_tools and library_tools.can_use_library_content(self)):
+            validation.set_summary(
+                StudioValidationMessage(
+                    StudioValidationMessage.ERROR,
+                    _(
+                        u"Only courses that are created using the split "
```

Comment:
@andy-armstrong @Kelketek Not sure how much it would help to add back the first sentence: "This course does not support content libraries. Only courses that are created using the split modulestore support content libraries". I think in earlier stories and in the documentation we avoided talking about split mongo vs old mongo, as users can't change what version they're using (if I understand correctly). I'm not sure if there are any actions a user could take in this case. Any?